scripts/west_commands/runners/nrfutil.py

```diff
@@ -115,6 +120,75 @@ def do_get_boards(self):
     def do_require(self):
         self.require('nrfutil')
 
+    def _generate_ncs_provision_key_file(
+        self,
+        keys: list[str] | str,
+        keyname: str,  # UROT_PUBKEY, BL_PUBKEY, APP_PUBKEY
+        output_file: Path
+    ):
+        """Generate a key file for ncs-provision
+
+        Currently uses the west ncs-provision command to generate the JSON file.
+        Consider reusing sdk-nrf/scripts/west_commands/ncs_provision.py
+        or sdk-nrf/scripts/generate_psa_key_attributes.py
```

Comment:
should probably mention that these are not part of the tree and only found in nrf-sdk